docs/conf.py

```diff
@@ -240,8 +240,20 @@ def generate_r_docs(app):
     cd {0}
     export R_LIBS="$CONDA_PREFIX/lib/R/library"
     Rscript build_r.R
-    Rscript build_r_site.R
-    """.format(os.path.join(CURR_PATH, os.path.pardir))
+    cd {1}
+    Rscript -e "roxygen2::roxygenize(load = 'installed')"
+    Rscript -e "pkgdown::build_site( \
+            lazy = FALSE \
```

Comment:
Should we explicitly pin `new_process` argument here?

Reply:
yeah probably a good idea. I will add that.